### PR TITLE
Chore: cleaning

### PR DESCRIPTION
**Pull request by haythem96**
# Task: cleaning

**Should be merged after #751**

## Description

Fixes ENG-810, ENG-831, ENG-830, ENG-818, ENG-811, ENG-841

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Document update

## How Has This Been Tested

Please describe how to test to verify the changes. Provide instructions so we can reproduce.

## FE Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] Added video recordings if it is a UI change

## User Facing Checklist

- [ ] I fully understand the user problem this PR is solving
- [ ] I know who the target user is for this PR and have a deep understanding of that user
- [ ] I have tried this flow thinking from the pov of the target user for this PR
- [ ] I (or working w someone on team) have scheduled a user test for this PR (if it is a large change)

### CODE REVIEW

File: packages/bull-vault/test/integration-test/AuctionBullTestFork.t.sol [hunk 1]
```diff
@@ -2023,6 +2023,250 @@ contract AuctionBullTestFork is Test {
         assertEq(squeethInCrab.add(wPowerPerpAmountToTrade), squeethInCrabAfter);
     }
 
+    function testFullRebalanceWhenEthUpAndCrTooHigh() public {
```
Comment (aleone) [inline packages/bull-vault/test/integration-test/AuctionBullTestFork.t.sol:2026]: i think we need a rebase, not reviewing this, think its in other PR

File: packages/bull-vault/test/integration-test/BullStrategyTestFork.t.sol [hunk 2]
```diff
@@ -192,7 +194,7 @@ contract BullStrategyTestFork is Test {
     }
 
     function testDepositWhenTotalSupplyLessThanMinimum() public {
-        uint256 crabToDeposit = 1e12;
+        uint256 crabToDeposit = WETH_DECIMALS_DIFF;
```
Comment (aleone) [inline packages/bull-vault/test/integration-test/BullStrategyTestFork.t.sol:197]: not supposed to be WETH_DECIMALS diff, should revert this change, just coincidentally 1e12

File: packages/bull-vault/test/integration-test/BullStrategyTestFork.t.sol [hunk 3]
```diff
@@ -276,7 +278,7 @@ contract BullStrategyTestFork is Test {
         _deposit(crabToDeposit);
         vm.stopPrank();
 
-        uint256 bullToRedeem = crabToDeposit.sub(1e12);
+        uint256 bullToRedeem = crabToDeposit.sub(WETH_DECIMALS_DIFF);
```
Comment (aleone) [inline packages/bull-vault/test/integration-test/BullStrategyTestFork.t.sol:281]: not supposed to be WETH_DECIMALS diff, should revert this change, just coincidentally 1e12

**Review by aleone — APPROVED** [review]
some 1e12s that were just quantities not WETH_DECIMAL_DIFF so should probably be reverted or changed to diff variable

**Comment by haythem96** [comment]
Need rebase before merging

**Review by alpinechicken — APPROVED** [review]
lgtm
